en/flight_modes/takeoff.md

```diff
@@ -32,21 +32,28 @@ Parameter | Description
 
 The aircraft takes off in the current direction using either *catapult/hand-launch mode* or *runway takeoff mode*.
 The mode defaults to catapult/hand launch, but can be set to runway takeoff using [RWTO_TKOFF](#RWTO_TKOFF).
+RC stick movement is ignored in both cases.
 
-In *catapult/hand launch mode* the vehicle will perform a full throttle climbout (ramp up to [RWTO_MAX_THR](#RWTO_MAX_THR) in about 2 seconds).
+### Catapult/Hand Launch {#hand_launch}
+
+In *catapult/hand launch mode* the vehicle waits to detect launch (based on acceleration trigger).
+On launch it ramps up to full throttle ([RWTO_MAX_THR](#RWTO_MAX_THR)) in about 2 seconds and then performs a full throttle climbout, with 10 degree takeoff pitch. 
 Once the altitude error < [FW_CLMBOUT_DIFF](#FW_CLMBOUT_DIFF), regular navigation will proceed.

```

Comment:
I think "regular navigation" is pretty ambiguous here.  I think what used to happen is that the fixed wing would loiter in Hold mode as soon as it reached the minimum loiter altitude (MIS_LTRMIN_ALT); does it still do this (I don't know)?

Reply:
@Antiheavy Yes, agreed. That is what I thought would happen too, but .... @RomanBapst can you confirm?

Reply:
Needs to be tested in navigation.  I think you are right though.

Reply:
@Antiheavy @hamishwillee I tested this in SITL and the result was different. Firstly, it's not straightforward to plan a mission with only a takeoff waypoint, PX4 mission feasibility checker requires you to have a landing pattern. So you should never get into the situation where there is just takeoff waypoint. However, I tricked the system and added a takeoff wapoint at the end of the mission and selected that as the current waypoint. The behavior is that the vehicles does a takeoff and then loiters about the takeoff waypoint at an altitude set by that waypoint.
MIS_LTRMIN_ALT did not have any effect.

Reply:
![image](https://user-images.githubusercontent.com/7610489/83491772-f3d64680-a4ba-11ea-894c-73ddfe1ec72e.png)

Reply:
In the upper figure you can see that the vehicles loiters at 18m about the takeoff waypoint. MIS_LTRMIN_ALT was set to 50.

Reply:
@RomanBapst actually we might be talking about 2 different takeoff modes.   There is Takeoff the flight mode and then their is an Automated takeoff in Mission mode using a Takeoff waypoint.  I think the logic and behavior is different.  You tested Mission mode takeoff above (which I think is how most people do it?).  This section of the Doc is talking about Takeoff the flight mode.  Admittedly it is confusing, and if you were confused I think the general users will be totally confused.

Reply:
@RomanBapst Todd's right - THIS doc is all about **takeoff mode**, not mission takeoff. So there is no takeoff height that appears to be set anywhere. I guess you have a target if this is triggered by a command protocol command, 
- but if you just set the mode, what height does it use?
- and does it go to loiter?